README.md

```diff
@@ -7,6 +7,7 @@
 <!-- /DARK_MODE_ONLY -->
 
 <p align="center">
+<img src="https://github.com/symforce-org/symforce/actions/workflows/ci.yml/badge.svg" />
```

Comment:
We should include an `alt` for this image.  I was also expecting to link to the actions page, like [black does](https://raw.githubusercontent.com/psf/black/main/README.md)?  Not sure if that's not recommended practice for some reason

Reply:
Good idea on both counts. I don't know why you wouldn't link to the actions page, so I'm gonna just do that.

Reply:
I ended up making it a link to https://github.com/symforce-org/symforce/actions/workflows/ci.yml. Could perhaps have it to be a link where the search is filtered particularly to the `main` branch (rather than simply all CI runs), but wasn't sure if it was bad practice to use a search link like https://github.com/symforce-org/symforce/actions/workflows/ci.yml?query=branch%3Amain

Reply:
Okay, per our in-person discussion, I've decided to link to the filtered search. It's more helpful this way, and if the link breaks somehow (like the link syntax for a filtered search changes - not sure if that's something that happens), then we can just update the link accordingly.